### PR TITLE
Create /status endpoint to check for recent notebooks

### LINKED ISSUE #62: Create a /status endpoint which checks if we have recent notebooks in our database
We want to monitor that our daily retrieval of notebooks is working. An external service will call this endpoint every day.

Currently, `StatusController` has an endpoint `/ok` which always returns `OK`.

In this issue, we want to create a new endpoint `/status`, in the same controller, which returns plain text:
- "OK" if there is at least one notebook in our database created in the last 49h.
- "ERROR. The most recent notebook was created on the #{notebook.inserted_at}".

### PR DESCRIPTION
**Pull request by dmarcoux**
This endpoint replaces the `/ok` endpoint.

Closes #62

### CODE REVIEW

File: lib/notesclub_web/controllers/status_controller.ex [hunk 1]
```diff
@@ -1,5 +1,17 @@
 defmodule NotesclubWeb.StatusController do
   use NotesclubWeb, :controller
 
+  import Notesclub.Notebooks, only: [get_latest_notebook: 0]
+
   def ok(conn, _params), do: text(conn, "OK")
```
Comment (dmarcoux) [inline lib/notesclub_web/controllers/status_controller.ex]: Should we remove this endpoint? I don't see the need for it anymore.

Comment (hectorperez) [reply lib/notesclub_web/controllers/status_controller.ex]: Thanks for this PR!

Sure, let's remove it. Let me know if you wish me to do it.

Comment (dmarcoux) [reply lib/notesclub_web/controllers/status_controller.ex]: I will do it, no worries 🙂

Comment (hectorperez) [reply lib/notesclub_web/controllers/status_controller.ex]: Thanks!

Comment (dmarcoux) [reply lib/notesclub_web/controllers/status_controller.ex]: Done, please have another look

File: lib/notesclub_web/router.ex [hunk 2]
```diff
@@ -44,7 +44,7 @@ defmodule NotesclubWeb.Router do
     pipe_through :browser
 
     # Used for uptime monitoring and zero-downtime deploys
```
Comment (hectorperez) [inline lib/notesclub_web/router.ex]: We can get rid of the comment. We now monitor `/` instead of `/ok` and deploys no longer need it.
```suggestion
```

**Review by hectorperez — COMMENTED** [review]
Looks good, thanks!

I only added a suggestion to remove an old comment.

File: lib/notesclub_web/router.ex (hunk 2 above)
Comment (dmarcoux) [reply lib/notesclub_web/router.ex]: Done!

**Pull request merged**